README.md

```diff
@@ -53,7 +53,8 @@ variables. These include the following:
  * `RASTER_CLUSTER_NAME`: The name of the Memberlist cluster.
  * `RASTER_ADVERTISE_MEMBERLIST_HOST`: The IP / hostname advertised by Memberlist.
  * `RASTER_ADVERTISE_MEMBERLIST_PORT`: The port advertised by Memberlist.
- * `RASTER_URL_SIGNING_SECRET`: A secret to use when validating signed URLs (optional).
```

Comment:
I personally really don't like the interface of having a thing1 and then a thing2 to say that you should pay attention to thing1. Ideally the presence of thing1 tells you that you should enable the feature. Personally I'd ditch `RASTER_URL_SIGNING_ENABLED` and just set a default value on `RASTER_URL_SIGNING_SECRET`. Docs, of course, should be updated to reflect the new reality.